spaces/aliases.json

```
@@ -23,5 +23,5 @@
 	"pepemon": "pepedontdump.eth",
 	"akropolis-delphi": "akropolis.eth",
 	"rally": "rallygov.eth",
-	"debaseonomics": "debaseonomics.eth"
+	"debaseonomics": "debaseonomics.eth",
```

Sorry a comma is still here! Please remove it as well.
```suggestion
	"debaseonomics": "debaseonomics.eth"
```